app/src/main/java/ch/hikemate/app/ui/map/RunHikeScreen.kt

```diff
@@ -613,3 +616,36 @@ private fun RunHikeBottomSheet(
         }
       }) {}
 }
+
+@Composable
+private fun ElevationGraphOnPercentage(
```

Comment:
Nitpick: This name is kinda weird. Why is the elevationgraph on the percentage? Maybe something like `ElevationGraphWithLocationPuck` or something like that.

Reply:
Agreed! 
Fixed